### PR TITLE
JENA-2222: Update of dependencies used in geosparql

### PR DESCRIPTION
**Pull request by afs**
This is "draft" initially.

It fixes the issue to make sure there is a fix available now.

It might be better to manage dependencies at the top level, partially or fully, or keep them specific to geosparql. Investigation needed.

### CODE REVIEW

**Comment by afs** [comment]
Second commit adds the dependencies into `dependenceManagement`.

File: jena-fuseki2/jena-fuseki-geosparql/pom.xml [hunk 1]
```diff
@@ -34,225 +34,206 @@
...
+      </dependency>
+
+      <dependency>
+        <!-- rdf-tables:1.0.4
+               -> com.opencsv:opencsv:jar:3.9 
+                 -> commons-beanutils:commons-beanutils:19.3
+             CVE-2019-10086 https://nvd.nist.gov/vuln/detail/CVE-2019-10086 
+             commons-beanutils:commons-beanutils:jar:1.9.3
+             Needs to be 1.9.4.
+             We exclude it from rdf-tables and diectly depend on it here.
```
Comment (kinow) [inline jena-fuseki2/jena-fuseki-geosparql/pom.xml:67]: s/diectly/directly

File: jena-fuseki2/jena-fuseki-geosparql/pom.xml [hunk 2]
```diff
@@ -34,225 +34,206 @@
...
+              <filter>
+                <artifact>*:*</artifact>
+                <excludes>
+                  <!-- Some jars are signed but shading breaks that.
+                       Don't include signing files.
+                  -->
+                  <exclude>META-INF/*.SF</exclude>
+                  <exclude>META-INF/*.DSA</exclude>
+                  <exclude>META-INF/*.RSA</exclude>
+                </excludes>
```
Comment (kinow) [inline jena-fuseki2/jena-fuseki-geosparql/pom.xml:220]: Whew, I can only imagine all the troubleshoot and documentation-reading needed to get all these configuration settings right @afs :point_down: :point_up: Thanks a lot!

**Review by kinow — APPROVED** [review]

File: jena-fuseki2/jena-fuseki-geosparql/pom.xml (hunk 2 above)
Comment (afs) [reply jena-fuseki2/jena-fuseki-geosparql/pom.xml:220]: There are certain patterns built-up over years, often by reading StackOverflow.

Does anyone ever write a POM file from scratch? 😃

**Pull request merged**